higlass/trackdata.py

```diff
@@ -0,0 +1,76 @@
+import slugid
+from higlass.client import Track
+from pathlib import Path
+from typing import List, Tuple, Union
+from numpy import cumsum
+
+import itertools as it
+from pathlib import Path
+
+def chromsize_pairs(chromsizes_fn):
+    with open(chromsizes_fn, 'r') as f:
+        chroms = [l.strip().split('\t') for l in f.readlines()]
+        chromnames = [c[0] for c in chroms]
+        chromlengths = [int(c[1]) for c in chroms]
+    
+    return list(zip(chromnames, chromlengths))
+
+def load_chromsizes(chrom_names_lengths):
+    chromnames, chromlengths = zip(*chrom_names_lengths)
+
+    cumlengths = list(it.accumulate(chromlengths))
+    # we want the cumlengths to start 0
+    cumlengths = [0] + cumlengths[:-1]
+
+    return dict([(name, {
+        'name': name,
+        'length': length,
+        'start': start
+    }) for name, length, start in zip(chromnames, chromlengths, cumlengths)])
+
+def bedtiles(lines: List[Tuple[str, int, int, str, str, str]],
+    chroms: Union[Path, str, list], **kwargs) -> Track:
+    """Generate a list of local tiles that can be used with a bedlike track.
```

Comment:
This seems to be raw sphinx docstrings instead of numpydoc. Shouldn't we be consistent?

Reply:
Good catch. We should indeed be consistent. I'll update the docstring.

It's was meant to be [Google style docstrings](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods).